partial-update/src/test/resources/partialupdate/GeneratedModel.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.autorest.partialupdate.util;
+
+import com.azure.core.annotation.Generated;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class Model {
```

Comment:
Is this right? The file name is `GeneratedModel.java` but the class name is `Model`. The file name and classname should match.